testing/features/support/components/footer/minimal.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Footer
+  class Minimal < ::Base
+    set_url "/iframe.html?id=components-footer--minimal&viewMode=story"
+
+    element :logo, ".cads-logo"
+    element :copyright, ".cads-footer__company-info div p:nth-of-type(1)"
+    element :company_info, ".cads-footer__company-info div p:nth-of-type(2)"
+
+    def validate_initial_state!
```

Comment:
the standard validation here should be used so we're not loading based off a validated assertion.

Reply:
There's a standard validation on `::Base` - So you can just remove this method.

Reply:
@celonicolosi We can remove this validation and just use the base one (And remove the footer element above)